### ISSUE TITLE
Consider removing merge tool

### ISSUE THREAD

**Issue by braughtg** [HFOSSedu/GitKit-Text#46]
The merge conflicts encountered are simple enough to resolve in the GitHub UI without using a merge tool.  It may be worth considering simplifying the activities to resolve the merge conflicts in that way.  This would not diminish the conceptual content of the activities, but would hopefully reduce excess overhead that seems to add confusion.  

It is easy to imagine adding additional activities later that would introduce more significant merge conflicts that would benefit from use of a GUI tool.

**Comment by rlampy** [comment]
I would agree with this thought. My experience with students and the merge tool was that some had issues getting it working, and most didn't see the benefit of using them (because the conflicts were simple). It would be useful to introduce the merge tool in an optional (or extra practice) section so they can see the directions but ignore until needed.

**Comment by braughtg** [comment]
Addressed by #93
